### PR TITLE
[README] add slack introduction

### PR DESCRIPTION
**Pull request by JingsongLi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,6 +55,15 @@ Paimon tracks issues in GitHub and prefers to receive contributions as pull requ
 
 <b style="color:red">Please make sure you are subscribed to the mailing list you are posting to!</b> If you are not subscribed to the mailing list, your message will either be rejected (dev@ list) or you won't receive the response (user@ list).
 
+## Slack
+
+You can join the Paimon community on Slack. Paimon channel is in ASF Slack workspace.
```
Comment (SteNicholas) [inline README.md:60]: Adds the link of ASF Slack workspace to help users to join.

Comment (liugddx) [reply README.md:60]: > Adds the link of ASF Slack workspace to help users to join.

I added a link for join Slack

[Get on Slack](http://bit.ly/3KYbORi)

Comment (SteNicholas) [reply README.md:60]: @liugddx, IMO, you could add a commit in this pull request. Meanwhile, line 60 could also add a link.

**Review by SteNicholas — APPROVED** [review]
LGTM.

**Pull request merged**